x/leverage/keeper/keeper.go

```diff
@@ -67,8 +67,8 @@ func (k Keeper) ModuleBalance(ctx sdk.Context, denom string) sdk.Int {
 // exchange for uTokens. If asset type is invalid or account balance is
 // insufficient, we return an error.
 func (k Keeper) LendAsset(ctx sdk.Context, lenderAddr sdk.AccAddress, loan sdk.Coin) error {
-	if !k.IsAcceptedToken(ctx, loan.Denom) {
-		return sdkerrors.Wrap(types.ErrInvalidAsset, loan.String())
+	if err := k.RequireLendEnabled(ctx, loan.Denom); err != nil {
```

Comment:
I think using the boolean logic is a bit cleaner than having a function return an error if false: 
```suggestion
	if !k.IsLendingEnabled(ctx, loan.Denom) {...
```

Reply:
I started with that function signature actually, but got persuaded by two points:
- I was immediately turning around to make an error "lending not enabled" on seeing the boolean
- It was no longer possible to distinguish "token does not exit" with "lending disabled" on boolean return

Reply:
Gotcha, makes sense with multiple codepaths - I'd suggest renaming this function to something like `[Check/Validate]LendingEnabled` rather than use the `Required` syntax then 😄

Reply:
_Note: Resolved the other two conversations to group them here - whatever conclusion we reach here will apply to RequireBorrowEnabled and RequireNotBlacklisted_

Reply:
Renamed all to assert - resolving conversation.